projects/plugins/jetpack/modules/module-headings.php

```diff
@@ -336,3 +336,943 @@ function jetpack_get_module_i18n_tag( $key ) {
 	}
 	return ! empty( $module_tags[ $key ] ) ? $module_tags[ $key ] : '';
 }
+
```

Comment:
This file already gets built by `jetpack build plugins/jetpack`. We should probably delete it from the repo, add it to `.gitignore`, and set `production-include` for it in `.gitattributes`. Much like we did for `modules/photon-cdn/jetpack-manifest.php` back in d4287fef9218c324de47fdf00621b680f8285a45 (part of #18313, which was part of #18257).

And probably also add it to the exclude list in `.phpcs.config.xml`, again like `modules/photon-cdn/jetpack-manifest.php` (#18400).